src/platform/plugins/shared/data/server/search/session/session_service.ts

```diff
@@ -162,11 +161,7 @@ export class SearchSessionService implements ISearchSessionService {
   };
 
   public find = async (
-    {
-      savedObjectsClient,
-      internalElasticsearchClient,
-      asCurrentUserElasticsearchClient,
-    }: SearchSessionStatusDependencies,
+    { savedObjectsClient, asCurrentUserElasticsearchClient }: SearchSessionStatusDependencies,
```

Comment:
Nit: Since we only use one client now, rename to `esClient` (as this is pretty ubiquitous in our code)